dockerfiles/harmoni/kinetic/full/dockerfile

```diff
@@ -76,6 +76,33 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
 RUN mkdir -p /root/local_mount/ \
     && ln -vs /root/harmoni_catkin_ws/src/HARMONI /root/local_mount/
 
+# ==================================================================
+# Download default models for local STT, TTS services
+# ------------------------------------------------------------------
+WORKDIR $ROS_WS/src/HARMONI/harmoni_models
+
+# STT
+RUN mkdir stt && cd stt \
+    && wget "https://github.com/mozilla/DeepSpeech/releases/download/v0.9.3/deepspeech-0.9.3-models.pbmm" \
+    && wget "https://github.com/mozilla/DeepSpeech/releases/download/v0.9.3/deepspeech-0.9.3-models.scorer"
+
+# TTS
+RUN cd .. \
+    && mkdir -p tts && cd tts \
+    && gdown --id 1dntzjWFg7ufWaTaFy80nRz-Tu02xWZos -O tts_model.pth.tar \
+    && gdown --id 18CQ6G6tBEOfvCHlPqP8EBI4xWbrr9dBc -O config.json \
+    && gdown --id 1Ty5DZdOc0F7OTGj9oJThYbL5iVu_2G0K -O vocoder_model.pth.tar \
+    && gdown --id 1Rd0R_nRCrbjEdpOwq6XwZAktvugiBvmu -O config_vocoder.json \
+    && gdown --id 11oY3Tv0kQtxK_JPgxrfesa99maVXHNxU -O scale_stats.npy
+
+WORKDIR $ROS_WS/src/HARMONI/harmoni_actuators/harmoni_tts
+
+RUN sudo apt-get update && sudo apt-get install espeak -y \
+    && git clone https://github.com/coqui-ai/TTS \
+    && cd TTS \
+    && git checkout b1935c97 \
+    && pip install -r requirements.txt \
+    && python setup.py install
```

Comment:
Just want to verify: have you built and run the changed dockerfiles?

Reply:
I have, and all the dependencies are set up correctly. However, I've found that I still need to run `catkin build` and `source devel/setup.bash` after entering the container so Python can import the DeepSpeechClient class. I noticed that `harmoni_entrypoint.sh` sources both ROS and devel `setup.bash` files but doesn't run `catkin build`; does that need to be added?

Reply:
I don't think we necessarily want to add it. The reason you need to run catkin build is because the version that is built with the container is on develop. Once this is merged into develop we can build and push the images with deepspeech already installed.